From ff2822f291e6a3b69012ed161554c5d46ee6f2fa Mon Sep 17 00:00:00 2001
From: Andy Whitcroft <apw@canonical.com>
Date: Tue, 17 Jul 2012 19:06:44 +0100
Subject: [PATCH] UBUNTU: SAUCE: highbank -- export clock functions for
 modules

Signed-off-by: Andy Whitcroft <apw@canonical.com>

TRIGGER 119a926bf04eae5024a3002b626ef8bc arch/arm/mach-highbank/clock.c
---
 arch/arm/mach-highbank/clock.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/mach-highbank/clock.c b/arch/arm/mach-highbank/clock.c
index c25a2ae..aed4724 100644
--- a/arch/arm/mach-highbank/clock.c
+++ b/arch/arm/mach-highbank/clock.c
@@ -27,24 +27,29 @@ int clk_enable(struct clk *clk)
 {
 	return 0;
 }
+EXPORT_SYMBOL(clk_enable);
 
 void clk_disable(struct clk *clk)
 {}
+EXPORT_SYMBOL(clk_disable);
 
 unsigned long clk_get_rate(struct clk *clk)
 {
 	return clk->rate;
 }
+EXPORT_SYMBOL(clk_get_rate);
 
 long clk_round_rate(struct clk *clk, unsigned long rate)
 {
 	return clk->rate;
 }
+EXPORT_SYMBOL(clk_round_rate);
 
 int clk_set_rate(struct clk *clk, unsigned long rate)
 {
 	return 0;
 }
+EXPORT_SYMBOL(clk_set_rate);
 
 static struct clk eclk = { .rate = 200000000 };
 static struct clk pclk = { .rate = 150000000 };
-- 
1.7.9.5

